src/inc/quic_platform_winuser.h

```diff
@@ -382,6 +382,133 @@ CxPlatPoolFree(
     }
 }
 
+#define CXPLAT_LOOKASIDE_MAXIMUM_DEPTH 4096
+
+typedef struct CXPLAT_LOOKASIDE CXPLAT_LOOKASIDE;
```

Comment:
Instead of all this, should we add an Ex function to initialize the `CXPLAT_POOL`?